python/hsfs/feature_view.py

```diff
@@ -76,16 +76,25 @@
 from hsml.model import Model
 
 
-_logger = logging.getLogger(__name__)
+if HAS_POLARS:
+    import polars as pl
 
-TrainingDatasetDataFrameTypes = Union[
-    pd.DataFrame,
-    TypeVar("pyspark.sql.DataFrame"),  # noqa: F821
-    TypeVar("pyspark.RDD"),  # noqa: F821
-    np.ndarray,
-    List[List[Any]],
-    pl.DataFrame,
-]
+    TrainingDatasetDataFrameTypes = Union[
+        pd.DataFrame,
+        TypeVar("pyspark.sql.DataFrame"),  # noqa: F821
+        TypeVar("pyspark.RDD"),  # noqa: F821
+        np.ndarray,
+        List[List[Any]],
+        pl.DataFrame,
+    ]
+else:
+    TrainingDatasetDataFrameTypes = Union[
+        pd.DataFrame,
+        TypeVar("pyspark.sql.DataFrame"),  # noqa: F821
+        TypeVar("pyspark.RDD"),  # noqa: F821
+        np.ndarray,
+        List[List[Any]],
+    ]
```

Comment:
Change this to:
```
TrainingDatasetDataFrameTypes = Union[
    pd.DataFrame,
    TypeVar("pyspark.sql.DataFrame"),  # noqa: F821
    TypeVar("pyspark.RDD"),  # noqa: F821
    np.ndarray,
    List[List[Any]],
]
if HAS_POLARS:
    import polars as pl
    TrainingDatasetDataFrameTypes.append(pl.DataFrame)
```

Reply:
I dont think we can do an append on an Union type. That is why I had it as two separate declarations

<img width="765" alt="image" src="https://github.com/user-attachments/assets/6a918438-0827-493f-90f9-835d6ddddd30">

Reply:
Oj, sorry. =) My bad. It should be:
```
TrainingDatasetDataFrameTypes = Union[TrainingDatasetDataFrameTypes, pl.DataFrame]
```